Frontend/src/app/query-interface/layout.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+import React, { useState } from "react";
+import PageWrapper from "@/layouts/page-wrapper";
+import { Layout, Menu, theme } from "antd";
+import type { MenuProps } from "antd";
+import { QueryInterfaceMenu } from "@/data/menu-data";
+import * as Routes from "@/routes/page-routes";
+import { useRouter } from "next/navigation";
+
+const { Content } = Layout;
+export default function GraphPanelLayout({
+  children,
+  params,
+}: Readonly<{
+  children: React.ReactNode;
+  params: { id: string };
+}>) {
+  const {
+    token: { colorBgContainer, borderRadiusLG },
+  } = theme.useToken();
+  const [current, setCurrent] = useState(Routes.QUERY_PANEL_ROUTES.query);
+  const router = useRouter();
+
+  const onClick: MenuProps["onClick"] = (e) => {
+    console.log("click ", e);
```

Comment:
No point of having the console log here.

Reply:
removed (added for debugging purpose)